extensions/wikia/Follow/FollowEmailTask.class.php

```diff
@@ -35,4 +33,16 @@ public function emailFollowNotifications( $initiatingUser, $aWatchers, $iUserId,
 			$oEmailNotification->notifyOnPageChange();
 		}
 	}
+
+	private function logWatchers( $watchers, $action ) {
+		if ( $action == FollowHelper::LOG_ACTION_BLOG_POST ) {
+			$msg = 'WatchlistLogs: Sending bloglisting watchlist updates';
+		} elseif ( $action == FollowHelper::LOG_ACTION_CATEGORY_ADD ) {
+			$msg = 'WatchlistLogs: Sending categoryadd watchlist updates';
+		} else {
+			$msg = 'WatchlistLogs: Sending other watchlist updates';
+		}
+
+		$this->info( $msg, [ 'watchedPages' => $watchers, 'action' => $action ] );
```

Comment:
I'm not sure about naming here. ```watchedPages``` vs ```watchers```. I know pages are use for keys but in values we still store watchers.